index.html

```diff
@@ -0,0 +1,464 @@
+                <a class="nav-link" href="#">
+                  Join
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  Sponsor
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  News
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4 red campaign">
+                <a class="nav-link" href="#">
+                  Dashboard
+                </a>
+              </li>
+            </ul>
+        </nav>
+      </div>
+    </header>
+
+
+    <main>
+      <!--jumbotron section-->
+      <section class="container-fluid jumbotron slider mb-0 pl-0 pr-0">
+        <div class="row">
+
+          <!--bars for small screens-->
+          <div class="pt-0 nav-small d-block d-md-none navbar-toggler" data-toggle="collapse" data-target="#navbarSupportedContent1">
+            <i class="fa fa-bars"></i>
+          </div>
+
+          <!--toggle between fa-bar and fa-times-->
+          <script>
+            $(".fa").click(function () {
+                $(".fa").toggleClass("fa-bars fa-times");
+            });
+          </script>
+
+          <!-- Collapsible content -->
+          <div class="collapse navbar-collapse" id="navbarSupportedContent1">
+
+            <!-- Links -->
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link red" href="./index.html">
+                  Home
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="./about.html">
+                  About
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  Tickets
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  Join
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  Sponsor
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4">
+                <a class="nav-link" href="#">
+                  News
+                </a>
+              </li>
+              <li class="nav-item pr-4 pl-4 red campaign">
+                <a class="nav-link" href="#">
+                  Dashboard
+                </a>
+              </li>
+            </ul>
+            <!-- Links -->
+
+          </div>
+          <!-- Collapsible content -->
+
+
+          <div class="welcome-text col-12 pt-5 mt-5">
+            <h3 class="text-left red">"Hello Chess Fans!"</h3>
+            <h1 class="image-effect text-uppercase text-left font-weight-bold red">Online Chess Conference</h1>
```

Comment:
This font is too large compared with the original design.
### Original design
![original-hero](https://user-images.githubusercontent.com/80149314/142493941-1629d6c5-cfd1-4e2e-a641-433828fed323.jpg)

### Your design
![review-hero](https://user-images.githubusercontent.com/80149314/142494029-e8b15ca7-2bc9-41a1-b5d7-a40ffe8af1b8.jpg)

Als o pay attention to the left margin and the start and end dates for the conference. Please make the necessary changes and send for another review.

Reply:
Hello Afani,

Thanks for your review :recycle: I have managed to reduce the size of the font on the jumbotron section and made changes to the margin. However I am not able to insert the dates as zetafonts don't allow input of 0 - 9 on their free trial of cocogoose font https://www.zetafonts.com/supportold&mvc=AUD.